core.coffee

```diff
@@ -36,6 +36,59 @@
 		'Options without $ prefixes (e.g. `options: filter: ...`) are deprecated, please use a $ prefix (eg `options: $filter: ...`).'
 	)
 
+	class Poll
+		channels:
+			'error': []
+			'data': []
+
+		stopped: false
+
+		intervalTime: 10000
+
+		constructor: (requestFn, intervalTime) ->
+			if typeof intervalTime is 'number'
```

Comment:
It would make sense to use `utils.isNumber` to be consistent with everywhere else in core.coffee